locale.c

```diff
@@ -5969,9 +5969,8 @@ Perl_sv_strftime_tm(pTHX_ SV * fmt, const struct tm * mytm)
                              );
     SV * sv = NULL;
     if (retval) {
-        STRLEN len = strlen(retval);
-        sv = newSV(len);
-        sv_usepvn_flags(sv, retval, len, SV_HAS_TRAILING_NUL);
+        sv = newSV(0);
```

Comment:
`Perl_sv_usepvn_flags` is going to immediately do `SvUPGRADE(sv, SVt_PV)`, calling `Perl_sv_upgrade`. Would be more efficient still to assign the SV using `sv = newSV_type(SVt_PV);`

Otherwise, LGTM.